### ISSUE TITLE
Standardized script for building a gem

### ISSUE THREAD

**Issue by sbellware** [eventide-project/contributor-assets#2]
To avoid packaging any symlinked libraries, stash the LIBRARIES_DIR first, then package, then replace the LIBRARIES_DIR.

**Comment by ntl** [comment]
I'm struggling to arrive at a means of stashing LIBRARIES_HOME.

If I were to, say, `mv $LIBRARIES_HOME ${LIBRARIES_HOME}-stashed`, it wouldn't actually solve the problem. Without getting into the details here (I can if you'd like), neither moving nor removing the LIBRARIES_HOME directory actually causes the symlinks nested inside other projects (such as `lib/consumer/postgres` and `lib/consumer/postgres.rb`) to go away.

I've got another hour to look at this, and if I can't figure it out, I'll post an update that reflects where I'm at.

**Comment by aaronjensen** [comment]
Can you assume git? If so, `git stash -a` would stash all files, including those ignored (but that would include your gems directory, which isn't great).

**Comment by ntl** [comment]
No, we cannot assume git @aaronjensen 

@sbellware I was able to write a script that removes all symlinks (including ones nested under e.g. `consumer` like `consumer/postgres.rb`. Stashing them is going to prove a bit more complicated, I'll have to devise a strategy for doing so reliably.

**Comment by sbellware** [comment]
@ntl Yeah, thinking through it in detail, it's more than I though it was.
